src/_data/sites/smg.json

```diff
@@ -0,0 +1,8 @@
+{
+    "url": "https://singlemotherguide.com",
+    "name": "Single Mother Guide",
+    "description": "A place for single mothers to feel a part of a larger community of women who are experiencing the challenges of single parenting.",
+    "twitter": "",
+    "authoredBy": ["Dawn"],
```

Comment:
These are Twitter accounts so if you don’t want a Twitter account associated with the site—I’d remove this line!

Reply:
Yes, you may remove the twitter handle for now @zachleat. Thank you.